--- a/src/w_wad.h
+++ b/src/w_wad.h
@@ -169,6 +169,7 @@ lumpnum_t W_CheckNumForName(const char *name);
 lumpnum_t W_GetNumForName(const char *name);
 lumpnum_t W_CheckNumForNamePwad(const char *name, UINT16 wad, UINT16 startlump);
 lumpnum_t W_GetNumForNamePwad(const char *name, UINT16 wad, UINT16 startlump);
+lumpnum_t W_GetNumForNameMusic(const char *name);
 
 size_t W_LumpLengthPwad(UINT16 wad, UINT16 lump);
 size_t W_LumpLength(lumpnum_t lumpnum);